docs/indyav/software/control.rst

```diff
@@ -0,0 +1,24 @@
+Control
+=======
+
+Control Packages
+----------------
+No software developed yet.
+
+Discussion
+-------------
+Initial development will start simple.
+
```

Comment:
Remove lines 4 - 11

Reply:
The headers define the template. I do not think they should be removed.